From e88a776a7bf5c7708ed0b67c92025e60d12824f1 Mon Sep 17 00:00:00 2001
From: Mike Banon <mikebdp2@gmail.com>
Date: Fri, 29 Nov 2019 17:50:26 +0000
Subject: [PATCH] Modify ./drivers/Makefile according to WR841N(D)v9 u-boot

---
 drivers/Makefile | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/Makefile b/drivers/Makefile
index 5a369df..fcaaa67 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -27,7 +27,8 @@ include $(TOPDIR)/config.mk
 
 LIB	= $(obj)libdrivers.a
 
-COBJS	= 3c589.o 5701rls.o ali512x.o atmel_usart.o \
+# cu570m start
+#COBJS	= 3c589.o 5701rls.o ali512x.o atmel_usart.o \
 	  bcm570x.o bcm570x_autoneg.o cfb_console.o cfi_flash.o \
 	  cs8900.o ct69000.o dataflash.o dc2114x.o dm9000x.o \
 	  e1000.o eepro100.o \
@@ -52,6 +53,9 @@ COBJS	= 3c589.o 5701rls.o ali512x.o atmel_usart.o \
 	  pxa_pcmcia.o mpc8xx_pcmcia.o tqm8xx_pcmcia.o	\
 	  rpx_pcmcia.o \
 	  fsl_i2c.o
+# cu570m
+COBJS	= pci.o
+# cu570m end
 
 SRCS	:= $(COBJS:.o=.c)
 OBJS	:= $(addprefix $(obj),$(COBJS))
-- 
2.17.1

