From e88a776a7bf5c7708ed0b67c92025e60d12824f1 Mon Sep 17 00:00:00 2001
From: Mike Banon <mikebdp2@gmail.com>
Date: Fri, 29 Nov 2019 17:50:26 +0000
Subject: [PATCH] Modify ./drivers/Makefile according to WR841N(D)v9 u-boot

---
 drivers/pci/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/pci/Makefile b/drivers/pci/Makefile
index fe45839466..70224dfbfb 100644
--- a/drivers/pci/Makefile
+++ b/drivers/pci/Makefile
@@ -25,12 +25,16 @@ include $(TOPDIR)/config.mk
 
 LIB 	:= $(obj)libpci.a
 
+ifeq ($(COMPRESSED_UBOOT),1) # cu570m condition branch
+COBJS-y += pci.o
+else # old default path
 COBJS-y += fsl_pci_init.o
 COBJS-y += pci.o
 COBJS-y += pci_auto.o
 COBJS-y += pci_indirect.o
 COBJS-y += tsi108_pci.o
 COBJS-y += w83c553f.o
+endif # cu570m
 
 COBJS	:= $(COBJS-y)
 SRCS 	:= $(COBJS:.o=.c)
-- 
2.24.0

