From 60c8fb818a5a58e0b1973f20b0104e7b4d524aed Mon Sep 17 00:00:00 2001
From: Mike Banon <mikebdp2@gmail.com>
Date: Fri, 29 Nov 2019 18:01:18 +0000
Subject: [PATCH] Modify ./tools/mkimage.c according to WR841N(D)v9 u-boot

---
 tools/mkimage.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/mkimage.c b/tools/mkimage.c
index 416e658..afe48cf 100644
--- a/tools/mkimage.c
+++ b/tools/mkimage.c
@@ -139,6 +139,7 @@ table_entry_t comp_name[] = {
     {	IH_COMP_NONE,	"none",		"uncompressed",		},
     {	IH_COMP_BZIP2,	"bzip2",	"bzip2 compressed",	},
     {	IH_COMP_GZIP,	"gzip",		"gzip compressed",	},
+    {   IH_COMP_LZMA,   "lzma",         "lzma compressed",      }, /* cu570m */
     {	-1,		"",		"",			},
 };
 
-- 
2.17.1

